packages/x-data-grid-pro/src/components/headerFiltering/GridHeaderFilterMenu.tsx

```diff
@@ -61,7 +65,20 @@ function GridHeaderFilterMenu({
         id={id}
         onKeyDown={handleListKeyDown}
       >
-        {operators.map((op, i) => {
+        {isApplied && [
+          <rootProps.slots.baseMenuItem
+            key="filter-menu-clear-filter"
+            iconStart={<rootProps.slots.columnMenuClearIcon fontSize="small" />}
```

Comment:
The menu with icons looks nice 👍, however if we could add icon per operator, like `=` for Equals and so on, and use focus state for currently active operator it'll look more visually appealing IMO.